webapi/src/test/scala/org/knora/webapi/routing/admin/ProjectsRouteZSpec.scala

```diff
@@ -39,332 +41,152 @@ object ProjectRouteZSpec extends ZIOSpecDefault {
   private val basePathProjectsShortcode: Path = !! / "admin" / "projects" / "shortcode"
 
   /**
-   * Test data and values used by multiple tests
-   */
-  // private val validProject: ProjectADM = ProjectADM(
-  //   id = "id",
-  //   shortname = "shortname",
-  //   shortcode = "AB12",
-  //   longname = None,
-  //   description = List(StringLiteralV2("description")),
-  //   keywords = List.empty,
-  //   logo = None,
-  //   ontologies = List.empty,
-  //   status = false,
-  //   selfjoin = false
-  // )
-
-  // private val validProject2: ProjectADM = ProjectADM(
-  //   id = "id2",
-  //   shortname = "shortname2",
-  //   shortcode = "AB13",
-  //   longname = None,
-  //   description = List(StringLiteralV2("description")),
-  //   keywords = List.empty,
-  //   logo = None,
-  //   ontologies = List.empty,
-  //   status = false,
-  //   selfjoin = false
-  // )
-
-  // private val expectedResponseNotFoundException: NotFoundException = NotFoundException("xxxx")
-
-  /**
-   * Common layers and expectations for ActorToZioBridge mock
-   */
-  // private val commonLayers: URLayer[ActorToZioBridge, ProjectsRouteZ] =
-  //   ZLayer.makeSome[ActorToZioBridge, ProjectsRouteZ](
-  //     AppConfig.test,
-  //     ProjectsRouteZ.layer,
-  //     RestProjectsService.layer,
-  //     authenticationMiddlewareLayer
-  //   )
```

Comment:
Do we still need all the commented out code in this file?

Reply:
not really. But when this PR is merged into the base branch (from our mob session) I'll have to do a clean-up pass anyways, so I think I'll do all the clean-up at once